pkg/ebpf/c/tracee.bpf.c

```diff
@@ -1904,7 +1907,7 @@ static __always_inline int init_program_data(program_data_t *p, void *ctx)
     p->event->task = (struct task_struct *) bpf_get_current_task();
     ret = init_context(ctx, &p->event->context, p->event->task, p->config->options);
     if (unlikely(ret < 0)) {
-        // disable logging as a workaround for instruction limit verifier error on kernel 4.19
+        // disable logging: workaround for inst limit verifier error on kernel 4.19
```

Comment:
I don't think shortening 'instruction' to 'inst' make it more readable, but less.